sdk/src/main/java/com/mapbox/maps/Style.kt

```diff
@@ -764,14 +834,23 @@ class Style internal constructor(
    * @return TRUE if and only if the style JSON contents, the style specified sprite and sources are all loaded, otherwise returns FALSE.
    */
   override fun isStyleLoaded(): Boolean {
-    return getStyleManager().isStyleLoaded
+    checkNativeStyle("isStyleLoaded")
+    return styleManager.isStyleLoaded
+  }
+
+  private fun checkNativeStyle(methodName: String) {
+    if (!isValid) {
+      Logger.e(TAG, "Accessing Style.$methodName after MapView is already destroyed meaning you are leaking Style object.")
```

Comment:
User does not operate (and most likely does not even know) what is a native map object. Leak could happen if `MapView` was already destroyed but user keeps on working with cached variable for `Style` or `MapboxMap` object and accessing methods. That's precisely what I tried to express.

Reply:
I mean - Style holds a lot of native memory, would be nice to explain it in more details, so that log message clearly said that : 
- Style/MapboxMap/etc should NOT be used after MapView is destroyed 
- It'll leak a lot of native memory in addition to Java object itself

Reply:
ok, I'll try to rephrase message text

Reply:
done